src/Discord.Net.Core/Entities/Channels/GuildChannelProperties.cs

```diff
@@ -24,6 +24,11 @@ public class GuildChannelProperties
         /// <summary>
         ///     Gets or sets the category ID for this channel.
         /// </summary>
+        /// <remarks>
+        ///     Setting this value to a category's snowflake identifier will change or set this channel's parent to the
+        ///     specified channel; setting this value to <c>0</c> will remove detach this channel from its parent if one
```

Comment:
```suggestion
        ///     specified channel; setting this value to <c>0</c> will detach this channel from its parent if one
```
Also, shouldn't this be `null`, not 0?